dss/storage/index_document.py

```diff
@@ -40,21 +46,30 @@ def add_to_index(self, index_name: str):
                             doc_type=ESDocType.doc.name,
                             id=str(self.fqid),
                             # FIXME: (hannes) Can this be json.dumps(self, indent=4) ?
-                            body=json.dumps(self))  # Don't use refresh here.
+                            body=self.to_json())  # Don't use refresh here.
         except Exception as ex:
             self.logger.error("Document not indexed. Exception: %s, Index name: %s, Index data: %s",
                               ex, index_name, json.dumps(self, indent=4))
             raise
         return initial_mappings
 
+    def to_json(self):
+        return json.dumps(self)
+
+    def __eq__(self, other: object) -> bool:
+        return (self is other
+                or super().__eq__(other)
```

Comment:
Please group these with parens. It's not immediately obvious what the priority/logic is here.

Reply:
Addressed this.